src/jshint.js

```diff
@@ -863,7 +863,7 @@ var JSHINT = (function() {
       funct["(blockscope)"].stack();
       advance("let");
       advance("(");
-      state.syntax["let"].fud.call(state.syntax["let"].fud, false);
+      state.syntax["let"].fud.call(state.syntax["let"].fud);
```

Comment:
seems like it was a common pattern - found lots of commits using it - https://github.com/jshint/jshint/commit/5924b2aa5aafdf8fede525b7156bd1962f824a14#diff-180beb5359ee54413144b3d8200d7504R3975

one thing is I didn't even notice it because its hard to compare & notice that `state.syntax["let"].fud` is repeated identically.

removing it doesn't have any effect, further notice that commit has a mix of different styles. I can't find a single committer introducing one style or the other.

Most of the other non-let/var/const don't use call. My vote would be to remove the call. I have an extra commit waiting, just say the word.